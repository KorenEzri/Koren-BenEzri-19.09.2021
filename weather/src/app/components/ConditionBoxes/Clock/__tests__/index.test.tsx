import * as React from 'react';
import { render } from '@testing-library/react';

import { Clock } from '..';


describe('<Clock  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Clock />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
