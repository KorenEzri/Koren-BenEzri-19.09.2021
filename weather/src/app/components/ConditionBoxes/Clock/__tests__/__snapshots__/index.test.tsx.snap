// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Clock  /> should match snapshot 1`] = `
.c2 {
  position: absolute;
  width: 3px;
  height: 8px;
  background: #262626;
  left: 0;
  right: 0;
  margin: 0 auto;
}

.c3 {
  position: absolute;
  width: 8px;
  height: 3px;
  background: #262626;
  top: 0;
  bottom: 0;
  right: 0;
  margin: auto 0;
}

.c4 {
  position: absolute;
  width: 3px;
  height: 8px;
  background: #262626;
  left: 0;
  right: 0;
  bottom: 0;
  margin: 0 auto;
}

.c5 {
  position: absolute;
  width: 8px;
  height: 3px;
  background: #262626;
  top: 0;
  bottom: 0;
  left: 0;
  margin: auto 0;
}

.c6 {
  height: 6px;
  width: 6px;
  position: absolute;
  left: 0;
  right: 0;
  top: 0;
  bottom: 0;
  margin: auto;
  background: #262626;
  border-radius: 100%;
}

.c7 {
  height: 200px;
  width: 200px;
  position: absolute;
  left: 60px;
  top: 60px;
  -webkit-transform: rotate(135deg);
  -ms-transform: rotate(135deg);
  transform: rotate(135deg);
  background: linear-gradient(transparent,rgba(#000,0.1));
}

.c8 {
  width: 3px;
  height: 100%;
  position: absolute;
  left: 0;
  right: 0;
  margin: 0 auto;
  -webkit-animation: time 60s infinite linear;
  animation: time 60s infinite linear;
}

.c8:before {
  position: absolute;
  content: '';
  background: #262626;
  height: 60px;
  width: 3px;
  top: 30px;
}

.c9 {
  width: 1px;
  height: 100%;
  position: absolute;
  left: 0;
  right: 0;
  margin: 0 auto;
  -webkit-animation: time 30s infinite linear;
  animation: time 30s infinite linear;
}

.c9:before {
  position: absolute;
  content: '';
  background: #262626;
  height: 40px;
  width: 1px;
  top: 50px;
}

.c10 {
  width: 2px;
  height: 100%;
  position: absolute;
  left: 0;
  right: 0;
  margin: 0 auto;
  -webkit-animation: time 15s infinite linear;
  animation: time 15s infinite linear;
}

.c10:before {
  position: absolute;
  content: '';
  background: #fd1111;
  height: 45px;
  width: 2px;
  top: 45px;
}

.c1 {
  margin-top: -5px;
  position: relative;
  height: 120px;
  width: 120px;
  background: white;
  box-sizing: border-box;
  border-radius: 100%;
  border: 10px solid #158176;
  position: absolute;
  margin-left: 40px;
}

.c0 {
  margin-top: -12px;
}

.c0 p {
  font-weight: bold;
  -webkit-letter-spacing: 1.5px;
  -moz-letter-spacing: 1.5px;
  -ms-letter-spacing: 1.5px;
  letter-spacing: 1.5px;
}

<div
  class="c0"
>
  <p>
    Invalid
  </p>
  <div
    class="c1"
  >
    <div
      class="c2"
    />
    <div
      class="c3"
    />
    <div
      class="c4"
    />
    <div
      class="c5"
    />
    <div
      class="c6"
    />
    <div
      class="c7"
    />
    <div
      class="c8"
    />
    <div
      class="c9"
    />
    <div
      class="c10"
    />
  </div>
</div>
`;
